src/KaggleSwagger.yaml

```diff
@@ -2041,30 +2041,67 @@ definitions:
         type: string
         description: The license that should be associated with the model instance
         default: Apache 2.0
+        # Use go/kaggle-models-license-values-swagger to get the accepted values (both abbreviation & full name are accepted)
```

Comment:
There is duplication since you need to update both for the update case and the create case. 

Given that @stevemessick is working towards using kapigen instead of Swagger, probably not worth the trouble of trying to optimize this now: https://docs.google.com/document/d/1WuLNP6XclSg0K_J8a1WPqtBPNWHOOX7uDi2CCuq6nEA